### ISSUE TITLE
Problem Sharing files (NextCloud 16 - Ubuntu 18.04 LTS) - Invalid query, please check the syntax. API specifications are here: http://www.freedesktop.org/wiki/Specifications/open-collaboration-services sharing

### ISSUE THREAD

**Issue by naxocaballero** [nextcloud/server#15740]
When I type an username to share or an external e-mail, I get the following error:

Invalid query, please check the syntax. API specifications are here: http://www.freedesktop.org/wiki/Specifications/open-collaboration-services sharing

And I can share files with nobody.
I don't know where to find the solution. And I think people don't have this issue.

Thank you.

**Comment by violoncelloCH** [comment]
could you please fill in the issue template with detailed information about your setup and how to reproduce your issue? happy to reopen your issue then :)

**Comment by gotmax23** [comment]
I have this same issue. I was just filling out my own new issue and noticed when I typed any occ command from the list like sudo -u www-data php occ app:list, it would read segmentation fault. I am on Ubuntu 18.04 on Digital Ocean installed using the web installer on Mariadb, PHP7.2, and Collabora in docker with a separate data folder. Here is my log https://cloud.saltz.com/index.php/s/KGEZ5ddzSa5ixPN. I also attached. 
[nextcloud.log](https://github.com/nextcloud/server/files/3230666/nextcloud.log)
Link sharing works properly, but sharing using a contact/email does not. Is this related to the segmentation fault? Client browser is Opera on Manjaro Linux.

Let me know if I should fill out my own issue. 

Thanks.
